src/main/java/com/conveyal/r5/analyst/cluster/AnalysisTask.java

```diff
@@ -81,6 +81,8 @@
      */
     public int nPathsPerTarget = 3;
 
+    public boolean logRequest = false;
```

Comment:
Please add Javadoc explaining what the parameter is for and how it is used in practice.